### PR TITLE
Create demos for nature components

### PR DESCRIPTION
**Pull request by alexkilgour**
Components done:
- `nature-footer`
- `nature-header`
- `nature-hero`
- `nature-section-heading`
- `nature-sort-by`

> **NOTE** On `nature-header` the protocol relative URLs have been removed as they are now [an antipattern](https://www.paulirish.com/2010/the-protocol-relative-url/) and they also break the demos

### Future work

A number of improvements have been identified in these components, and some areas that need more thought. These have been collected in the following issues - #667 #668 #669 #670 #671

### CODE REVIEW

File: toolkits/nature/packages/nature-sort-by/README.md [hunk 1]
```diff
@@ -37,57 +50,9 @@ This component uses Global Expander. This means you should be aware of the follo
...
-                <input name="order" id="orderdate_asc" value="date_asc" type="radio">
-                <label for="order-date_asc">
-                    <span>Date — oldest first</span>
-                </label>
-            </li>
-        </ul>
-    </div>
-</div>
-```
+A configurable template can found in the [`view` folder](./view/sort-by.hbs). Example usage can be found in the `demo` folder.
```
Comment (amyhupe) [inline toolkits/nature/packages/nature-sort-by/README.md:55]:
```suggestion
Find a configurable template in the [`view` folder](./view/sort-by.hbs). You can see an example usage in the `demo` folder.
```

**Review by amyhupe — COMMENTED** [review]
Hi Alex, a few suggestions, comments and a couple of questions for you here. Hope all makes sense.

File: toolkits/nature/packages/nature-footer/README.md [hunk 2]
```diff
@@ -1,44 +1,36 @@
...
+@import '@springernature/brand-context/default/scss/60-utilities/hiding.scss';
+@import '@springernature/brand-context/default/scss/60-utilities/icons.scss';
+@import '@springernature/brand-context/default/scss/60-utilities/spacing.scss';
+```
+
+> **NOTE** The component require the use of the utility classes shown above
+
+## Template
+
+A configurable template can found in the [`view` folder](./view/footer.hbs). Example usage can be found in the `demo` folder.
```
[the remark this answers is not in the document]
Comment (alexkilgour) [reply toolkits/nature/packages/nature-footer/README.md:33]: Yes, this should now render the demo on the website

File: toolkits/nature/packages/nature-footer/demo/context.json [hunk 3]
```diff
@@ -0,0 +1,131 @@
+{
+  "logo": {
+    "name": "nature-portfolio",
+    "width": "200",
+    "height": "31"
+  },
+  "title": "brand sitemap",
+  "footerLinks": [
+    {
+      "uri": "https://www.nature.com/npg_/company_info/index.html",
```
[the remark this answers is not in the document]
Comment (alexkilgour) [reply toolkits/nature/packages/nature-footer/demo/context.json:10]: This is [meant to be URI](https://danielmiessler.com/study/difference-between-uri-url/) as this follows the format already in use on our existing applications. It just so happens that in this case the URI is a URL!

[the remark this answers is not in the document]
Comment (amyhupe) [reply toolkits/nature/packages/nature-footer/demo/context.json:10]: Ah OK - thanks for explaining!

**Review by jon-stevens — APPROVED** [review]

**Pull request merged**